src/path_manager.c

```diff
@@ -742,28 +752,56 @@ static void handle_mptcp_event(struct l_genl_msg *msg, void *user_data)
 /**
  * @brief Verify that MPTCP is enabled at run-time in the kernel.
  *
- * Mptcpd requires MPTCP to be enabled in the kernel at run-time.
- * Issue a warning if it is not enabled.
+ * Check that MPTCP is supported in the kernel by opening a socket
+ * with the IPPROTO_MPTCP protocol.
```

Comment:
detail: should we add a comment here or in the function name to precise that here we check if we have "MPTCP Upstream" and below, it is the multipath-tcp.org version?

Reply:
Sure.  That's a good idea.